src/proto/grpc/testing/xds/v3/common.proto

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Local copy of Envoy xDS proto file, used for testing only.
+
+syntax = "proto3";
+
+package envoy.extensions.transport_sockets.tls.v3;
+
+import "src/proto/grpc/testing/xds/v3/string.proto";
+
+message CertificateValidationContext {
```

Comment:
We can probably just put this message in tls.proto instead of creating another file.  The name common.proto doesn't really make sense without the original directory structure.

Reply:
ack, Done!